src/utils/changelog.ts

```diff
@@ -214,8 +215,10 @@ interface Commit {
   author?: string;
   hash: string;
   title: string;
+  body: string;
   hasPRinTitle: boolean;
   pr: string | null;
+  prBody?: string | null;
```

Comment:
is this used anywhere? seems like it is always undefined

Reply:
Comes from the spread GitHub API response somewhere in the code.